Lorem Ipsum XR Translator/Assets/Scripts/SpeechToText/AzureSpeechToText.cs

```diff
@@ -0,0 +1,75 @@
+using UnityEngine;
+using Microsoft.CognitiveServices.Speech;
+using Config;
+using System.Threading.Tasks;
+
+namespace STT
+{
+    /// <summary>
+    /// Class AzureSpeechToText in STT namespace is used to convert Speech to text using Azure speech SDK
+    /// </summary>
+    public class AzureSpeechToText : MonoBehaviour
+    {
+        /// <summary>
+        /// string serviceRegion represents the region selected in the azure services
+        /// </summary>
+        private string serviceRegion = "northeurope";
+
+        /// <summary>
+        ///SSTRequest() is a asynchronous method  and returns a Task<string>. This means when you call this method,
+        ///it will run asynchronously, and once it's done, it will return a string.
+        /// </summary>
+        public async Task<string> SSTRequest()
+        {
+            /// <summary>
+            /// This debug log provides a message when the Button is pressed
+            /// </summary>
+            Debug.Log("Button was pressed.");
```

Comment:
This one can be removed since we're no longer tying this service to a 'button.'

Reply:
removed